package.json

```diff
@@ -70,6 +89,10 @@
       "node",
       "browser"
     ],
+    "ignores": [
```

Comment:
Is regular XO even used anymore?

Reply:
I got XO working using @typescript-eslint, you can check it at my typescript PR in evergreen